Failing doctests

The doctests started failing today with

```
Warning, treated as error:
Invalid configuration value found: 'language = None'. Update your configuration to a valid langauge code. Falling back to 'en' (English).
make: *** [Makefile:50: html] Error 2
make: Leaving directory '/home/runner/work/datalad/datalad/docs'
Error: Process completed with exit code 2.
```

This is due to [today's release of Sphinx 5.0](https://www.sphinx-doc.org/en/master/changes.html#release-5-0-0-released-may-30-2022) which comes with https://github.com/sphinx-doc/sphinx/issues/10474 that disallows the
``language = None`` configuration we currently still use.

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -120,7 +120,7 @@ release = datalad.__version__
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
 
 # There are two options for replacing |today|: either, you set today to some
 # non-false value, then it is used:

